integration/tests/cook/test_basic.py

```diff
@@ -384,7 +384,6 @@ def test_docker_fields(self):
             self.assertIn({'mode': 'RW',
                            'host-path': '/var/lib/mno',
                            'container-path': '/var/lib/pqr'}, volumes)
-            util.wait_for_job(self.cook_url, job_uuid, 'completed')
```

Comment:
Maybe we should have a wait for job that doesn't assert on the status, just the the job stopped running. Removing the wait may otherwise have implications on the number of jobs running concurrently from the tests.

Reply:
This `wait_for_job` was recently added, but then we ended up deciding to create a completely separate test for making sure that the job completes with some of these parameters. This test is just to check that the fields got set correctly.